extensions/sql-migration/src/wizard/migrationModePage.ts

```diff
@@ -26,8 +26,14 @@ export class MigrationModePage extends MigrationWizardPage {
 	}
 
 	public async onPageEnter(): Promise<void> {
+		this.wizard.registerNavigationValidator((e) => {
+			return true;
```

Comment:
Are these just to clear the previous navigation validators?

Reply:
Yes